File: backend/src/admin/objects/OverviewResponse.scala
package delivery.admin.objects

import delivery.merchant.objects.Merchant
import delivery.order.objects.Order
import delivery.rider.objects.Rider

final case class OverviewResponse(
    merchants: List[Merchant],
    orders: List[Order],
    riders: List[Rider],
    campaigns: List[PromotionCampaign],
    complaintTickets: List[ComplaintTicket]
)

File: frontend/src/objects/admin/OverviewResponse.ts
import type { Merchant } from '@/objects/merchant/Merchant'
import type { Order } from '@/objects/order/Order'
import type { Rider } from '@/objects/rider/Rider'
import type { PromotionCampaign } from './PromotionCampaign'
import type { ComplaintTicket } from './ComplaintTicket'

export interface OverviewResponse {
  merchants: Merchant[]
  orders: Order[]
  riders: Rider[]
  campaigns: PromotionCampaign[]
  complaintTickets: ComplaintTicket[]
}
